packages/atlas/src/providers/uploadsManager/uploadsManager.tsx

```diff
@@ -70,8 +69,7 @@ export const UploadsManager: React.FC = () => {
             description: video.parentObject?.title || '',
             iconType: 'success',
             timeout: UPLOADED_SNACKBAR_TIMEOUT,
-            actionText: 'See on Joystream',
-            actionIcon: <SvgActionNewTab />,
+            actionText: 'Watch',
```

Comment:
Can you provide source of this change? I cannot find it in figma

Reply:
In the first version of the snack bar, the action button was placed on the right, close to the close button and there wasn't a lot of space for longer text and icons. That's why I initially changed this. 
This is how designers wanted snakcbars to look initially: 
![image](https://user-images.githubusercontent.com/51168865/151754369-2777fa61-6d29-4e68-9ea5-aa9ba04a49c4.png)

Anyway, they changed the design again in #2005, so I'll just revert this change ;) 
![image](https://user-images.githubusercontent.com/51168865/151754531-734c7d48-af46-48df-a073-aee9596adf2b.png)